src/Assignments/ProblemIframe.tsx

```diff
@@ -382,11 +389,17 @@ export const ProblemIframe: React.FC<ProblemIframeProps> = ({
                     ww_applet_list[key].submitAction();
                     problemForm.dispatchEvent(new Event('input'));
                 }, 100, {leading:true, trailing:true}));
-            }); 
-            await Promise.all(promises);       
+            });
+            await Promise.all(promises);
         }
 
-        setLoading(false);
+        setRenderedHTML((renderedHTML: string): string => {
+            if (renderedHTML !== '') {
+                iframeDoc?.dispatchEvent(new Event('Rederly Loaded'));
```

Comment:
I gave this string as an example, I'm fine with it but I don't know if matches conventions (with the caps and the space)